Doc/whatsnew/3.9.rst

```diff
@@ -105,6 +105,15 @@ Merge (``|``) and update (``|=``) operators have been added to the built-in
 :class:`dict` class.  See :pep:`584` for a full description.
 (Contributed by Brandt Bucher in :issue:`36144`.)
 
+removeprefix and removesuffix methods for strings
```

Comment:
In What's New in Python 3.8, PEP numbers are not well advertised. I'm not sure if it's intentational or not. Maybe add it in the title?

```suggestion
PEP 616: New removeprefix() and removesuffix() string methods
```